pkg/cmd/policy_validate.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pulumi/pulumi/pkg/backend"
+	"github.com/pulumi/pulumi/sdk/go/common/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyValidateCmd() *cobra.Command {
+	var argConfig string
+
+	var cmd = &cobra.Command{
+		Use:   "validate <org-name>/<policy-pack-name> <version>",
+		Args:  cmdutil.ExactArgs(2),
+		Short: "[PREVIEW] Validate Policy Pack configuration against config schema",
+		Long:  "Validate Policy Pack configuration against the specified version of its configuration schema.",
```

Comment:
Maybe: "Validate a Policy Pack configuration against the configuration schema of the specified version."

Reply:
agreed. i think that sounds better!